### PR TITLE
Inclusive naming round 2

### PR DESCRIPTION
**Pull request by tahliar**
### Description

Fixed additional language based on https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Explained/singlehtml/index.html (SP4 only).

### Backports

- [x] To maintenance/SLEHA15SP4
- [ ] To maintenance/SLEHA15SP3
- [ ] To maintenance/SLEHA15SP2
- [ ] To maintenance/SLEHA15SP1
- [ ] To maintenance/SLEHA15
- [ ] To maintenance/SLEHA12SP5
- [ ] To maintenance/SLEHA12SP4

### References

jsc#SLE-22017

### CODE REVIEW

File: xml/ha_config_cli.xml [hunk 1]
```diff
@@ -1775,13 +1774,13 @@ hostname (string): Hostname
...
     op monitor interval=60 \
-    op monitor interval=61 role=Master</screen>
+    op monitor interval=61 role="Promoted</screen>
      </step>
      <step>
       <para>
        Create the promotable clone resource:
       </para>
-<screen>&prompt.crm.conf;<command>ms</command> ms-rsc my-rsc</screen>
+<screen>&prompt.crm.conf;<command>clone</command> clone-rsc my-rsc</screen>
```
Comment (gao-yan) [inline xml/ha_config_cli.xml:1783]: Hi @tahliar , I just noticed we are missing `meta promotable=true` here. So basically it should be:

`clone clone-rsc my-rsc meta promotable=true`

Or:
```
clone clone-rsc my-rsc \
    meta promotable=true
```

Comment (tahliar) [reply xml/ha_config_cli.xml:1783]: Thanks @gao-yan! I've fixed that in #272.

File: xml/ha_config_cli.xml [hunk 2]
```diff
@@ -1775,13 +1774,13 @@ hostname (string): Hostname
       </para>
 <screen>&prompt.crm.conf;<command>primitive</command> my-rsc ocf:myCorp:myAppl \
     op monitor interval=60 \
-    op monitor interval=61 role=Master</screen>
+    op monitor interval=61 role="Promoted</screen>
```
Comment (gao-yan) [inline xml/ha_config_cli.xml:1777]: We could drop the single quotation mark here :-)

Comment (tahliar) [reply xml/ha_config_cli.xml:1777]: Oh dear! Thanks for spotting it! :sweat_smile:

**Pull request merged**